### ISSUE TITLE
BRIG keeps generating SYS_ERROR:org.xml.sax.SAXException: value for 'sequenceLength' attribute in cgview element must be less than 20000000 in null at line 1 column

### ISSUE THREAD

**Issue by com31** [happykhan/BRIG#14]
![Screenshot from 2019-12-09 16-20-04](https://user-images.githubusercontent.com/25859908/70430106-0e95a880-1aa0-11ea-8d22-24f4689b7cde.png)
Can anyone helped me with this issue

**Comment by happykhan** [comment]
Hi.


Your reference is huge. Over 20MB.

**Comment by daniel251994** [comment]
Thank you for your reply..........anyways so the size of the reference genme should always be <20 MB ?

**Comment by sunnyEV** [comment]
Thanks for this post,

I'm plotting the figure for comparative mito-genome study.

My db is 1 GB, how do I run a BRIG then ?

**Comment by happykhan** [comment]
@daniel251994 Yes the ref genome should be under <20MB that is a hard limit by CGVIEW. 

I

**Comment by happykhan** [comment]
@sunnyEV BRIG is likely not appropriate for what you are doing.
